src/app/modules/item/pages/item-details/item-details.component.ts

```diff
@@ -3,20 +3,30 @@ import { CurrentContentService, PageInfo } from 'src/app/shared/services/current
 import { ActivatedRoute } from '@angular/router';
 import { itemFromDetailParams, NavItem, isPathGiven } from 'src/app/shared/services/nav-types';
 import { GetBreadcrumbService, BreadcrumbItem } from 'src/app/modules/item/http-services/get-breadcrumb.service';
-import { EMPTY, Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
+import { EMPTY, forkJoin, Observable, of, throwError } from 'rxjs';
+import { map, mapTo, switchMap } from 'rxjs/operators';
+import { ItemTabService } from '../../services/item-tab.service';
+import { GetItemByIdService } from '../../http-services/get-item-by-id.service';
+import { ResultActionsService } from 'src/app/shared/http-services/result-actions.service';
 
 @Component({
   selector: 'alg-item-details',
   templateUrl: './item-details.component.html',
-  styleUrls: ['./item-details.component.scss']
+  styleUrls: ['./item-details.component.scss'],
+  providers: [ ItemTabService ]
 })
 export class ItemDetailsComponent implements OnDestroy {
 
+  item$ = this.itemTabService.item$;
+  state: 'loaded'|'loading'|'error' = 'loading';
```

Comment:
`loaded` and `loading` are very similar, maybe change `loaded` to `ready` or something different

Reply:
I will change it anyway in a next PR because the whole thing is not really correct.